### DISCUSSION TITLE
RAG context obfuscation for web searches

### DISCUSSION THREAD

**Discussion by raydoomed** [open-webui/open-webui#7392] [General]
I found that in a continuous conversation, the answer to the second web search would confuse the time based on the context results. For example, when I asked who someone was for the first time, his answer was correct. When I changed the keywords the second time, he always answered based on his knowledge base data and did not answer based on the online content. It was always data before 2023. How can I solve this problem?

**Comment by mikygit** [comment]
Same here i find the web search tool bery unstable. There are many other discussions around it looking for a solution / explanation.

**Reply by raydoomed** [reply]
![Screenshot_20241216_021353_Vivaldi.jpg](https://github.com/user-attachments/assets/8c775e94-92cf-4170-885b-b5b1b06db9d4)

I seem to have solved this problem. The problem is not in the search engine, and the model needs to be set, as shown in the figure.
